include/dlaf/eigensolver/bt_reduction_to_band/impl.h

```diff
@@ -162,6 +163,14 @@ void BackTransformationReductionToBand<backend, device, T>::call(
   dlaf::matrix::Distribution dist_t({mb, total_nr_reflector}, {mb, mb});
   matrix::Panel<Coord::Row, T, device> panelT(dist_t);
 
+  const auto dist_ws = [&]() {
+    using dlaf::factorization::internal::getTFactorNWorkers;
+    const SizeType nworkspaces = to_SizeType(std::max<std::size_t>(0, getTFactorNWorkers() - 1));
```

Comment:
Naive question: 0 workspaces is ok? I guess this is the case when you reduce directly into the final tile without any intermediate workspaces, and the number of workers is 1?

Reply:
Yes, exactly.

There is another point to raise here: it is not said that all workspaces allocated will be used. It is legit to think that some are not going to used always (e.g. the panel is going to shrink in size over iterations so less workers are enough), but it might also happen that some tiles are never going to be used because the biggest size of the panel is not enough to have all the workers involved.

I preferred to not involve any logic about how many workers are actually going to be involved and kept on the safe side allocating one tile for any possible worker thread that might ever be involved given the configuration.

I might see the argument about possible memory waste, but at the same time I don't see it as a threatening or compelling topic.